README.md

```diff
@@ -22,24 +22,24 @@ Book 1-----n Author
 This is how related resources are loaded by default using embedded mode.
 
 ```json
-{  
```

Comment:
Just removal of trailing spaces, sorry, only real change here is that for Laravel support a new version would be needed as some of the dependencies updated are not compatible with both Laravel 5 and 6